src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its engine and began to work.");
```

Comment:
The message 'Truck started its engine and began to work.' could be more informative by specifying that it's a Truck that is working. Consider rephrasing it to include the type of Machine, for example: 'Truck is working.'